schema.graphql

```diff
@@ -1361,7 +1361,7 @@ enum InstructionStatusEnum {
 
 type Leg @entity {
   id: ID! # instructionId/legIndex
-  asset: Asset!
+  assetId: String! # @prashantasdeveloper Replace this with `asset: Asset` when chain validations are fixed
```

Comment:
Could this be `Asset`, as in nullable reference?

Even when chain validations are in place we still need to support the potential references made before then.

Reply:
We can change this to support null when we have that in place. For now lets stick to it